packages/core/src/Chip/FilterChip/FilterChip.tsx

```diff
@@ -12,6 +12,7 @@ const propTypes = {
   ...space.propTypes,
   ...fontSize.propTypes,
   id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
```

Comment:
I'm not sure if we should require the new prop. Even though it doesn't break functionality, it's technically a breaking change because the change to the API isn't purely additive if we add a required prop. What do you think?

Reply:
Per the discussion with @craigpalermo @sdalonzo and @BeniCheni, the final agreement is to mark the property as resolved.